neuro-sdk/src/neuro_sdk/_disks.py

```diff
@@ -85,10 +87,22 @@ def _get_disks_url(self, cluster_name: Optional[str]) -> URL:
         return self._config.get_cluster(cluster_name).disks_url
 
     @asyncgeneratorcontextmanager
-    async def list(self, cluster_name: Optional[str] = None) -> AsyncIterator[Disk]:
+    async def list(
+        self,
+        cluster_name: Optional[str] = None,
+        org_name: Optional[str] = None,
+        project_name: Optional[str] = None,
+    ) -> AsyncIterator[Disk]:
         url = self._get_disks_url(cluster_name)
+        params = {}
+        org = org_name or self._config.org_name
+        if org:
+            params["org_name"] = org
+        if project_name:
+            params["project_name"] = project_name
```

Comment:
this might seem a bit inconsistent in terms of other methods where we fall back to `_config` if `project_name` is missing.

in general, using `cluster_name` as is but falling back to `org_name` and `project_name` seems a bit off, but we probably don't want to dive into this now.

Reply:
Yes, let's agree on the behavior for those operations and create a corresponding issue to tackle it separately, wdyt?

Here I followed the reasoning: creating resources always requires `project_name` so we fall back. The listing does not require it, so we are safe to list everything available for the user in favor of UX (unless we filter results). As for the update, if the resources could not be identified (referred by name, not ID), we fall back.

Reply:
added https://github.com/neuro-inc/neuro-cli/issues/2962